src/lib/helpers/getPrerenderManifest.js

```diff
@@ -26,9 +26,11 @@ const transformManifestForI18n = (manifest) => {
   return { ...manifest, routes: newRoutes }
```

Comment:
not for now, but if we Promise.all these we'll get slightly better perf

```js
const promises = items.map(async (item) => {
  return await thing(item);
})

await Promise.all(items);

// continue
```

not a dealbreaker by any means, but something that might help us eke some extra ms out of this